clickhouse_test.go

```diff
@@ -0,0 +1,123 @@
+package goose
+
+import (
+	"database/sql"
+	"os/exec"
+	"strings"
+	"testing"
+
+	_ "github.com/kshvakov/clickhouse"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClickHouseDialect(t *testing.T) {
+	out, err := exec.Command("go", strings.Fields("build -i -o ./cmd/goose/goose ./cmd/goose")...).CombinedOutput()
```

Comment:
Why do we build goose from within Go tests?

Reply:
Because you already do it in your tests https://github.com/pressly/goose/blob/master/goose_test.go#L9

Reply:
That was probably a temporary to have at least some tests :) We shouldn't do that in Go tests :)

Reply:
Can we remove dependency on github.com/stretchr/testify/assert?